documentation/src/main/asciidoc/ch11.asciidoc

```diff
@@ -825,7 +860,7 @@ validator factory still is referenced by application code.
 ====
 
 [[section-time-provider]]
-=== Time providers for `@Future` and `@Past`
+=== Time providers for @Future and @Past
```

Comment:
> Does it look bad with the mark-up in titles?

+1 Default rendering sucks with back-tics in the title. I hope asciidoctor will give you soon more control over how reference format and numbering looks